main/src/cgeo/geocaching/utils/SystemInformation.java

```diff
@@ -102,7 +103,7 @@ public static String getSystemInformation(@NonNull final Context context) {
         if (GCConnector.getInstance().isActive()) {
             body.append("\n- Geocaching.com date format: ").append(Settings.getGcCustomDate());
         }
-        body.append("\n- BRouter connection available: ").append(Routing.isAvailable());
+        body.append("\n- BRouter installed: ").append(ProcessUtils.isInstalled(context.getString(R.string.package_brouter))).append(" / connection available: ").append(Routing.isAvailable());
```

Comment:
Just to understand the difference;
What is the trigger, when `Routing.isAvailable` becomes true? The user actively uses it (by setting a navigation method other than straight) or something else?

Reply:
`Routing.isAvailable()` becomes true on connecting to the BRouter service, which gets triggered by opening the map (independent from routing mode).